### ISSUE TITLE
User token with the react-todo example

### ISSUE THREAD

**Issue by tatecarson** [textileio/js-examples#13]
I am attempting to get the react-todo example running. I realized that it has a hardcoded token. What code do I need to add to get this example to work? 

I have the CLI setup and have created account and user keys but I am not sure what to do next.

**Comment by carsonfarmer** [comment]
That example has not yet been updated. We are hoping to get that example back up to date very soon!

**Comment by tatecarson** [comment]
Which examples are updated and working? I really want to try out textile but it is a bit confusing getting started.

**Comment by carsonfarmer** [comment]
Yes, we’re a bit behind on the example updates and docs. Trying to get them all updated is taking time. See the readme for this repo for the list of updated examples.
